GameServerLib/GameObjects/Stats/Stats.cs

```diff
@@ -214,23 +214,22 @@ public void LevelUp()
         {
             Level++;
 
-            HealthPoints.BaseValue += HealthPerLevel;
-            CurrentHealth = HealthPoints.Total / (HealthPoints.Total - HealthPerLevel) * CurrentHealth;
-            ManaPoints.BaseValue = ManaPoints.Total + ManaPerLevel;
-            CurrentMana = ManaPoints.Total / (ManaPoints.Total - ManaPerLevel) * CurrentMana;
-            AttackDamage.BaseValue += AdPerLevel;
-            Armor.BaseValue += ArmorPerLevel;
-            MagicResist.BaseValue += MagicResistPerLevel;
-            HealthRegeneration.BaseValue += HealthRegenerationPerLevel;
-            ManaRegeneration.BaseValue += ManaRegenerationPerLevel;
-
+            StatsModifier statsLevelUp = new StatsModifier();
+            statsLevelUp.HealthPoints.BaseBonus = HealthPerLevel;
```

Comment:
Should it be `=` or did you mean `+=`? (as all the others are `+=`)
Alternatively, shouldn't all the others be `=` as you only declare the variable? (I assume the default value is 0?)

Reply:
Oh, it was supposed to be just =
Functionally = or += doesn't change anything in this case, but I'll be uploading the changes anyway.